tests/converter/test_df_converter.py

```diff
@@ -93,5 +93,25 @@ def test_df_converter():
         "All right. So, you're transferring $200 from your checking to a savings. Is that right?"
     )
 
+    conv.process("hello!")
+    conv.assert_text(
```

Comment:
is it easier to assert domain/intent here?

Reply:
I think its fine to assert the whole NLR since we want to test if the output behavior is consistent for a given DF app